src/coins.ts

```diff
@@ -833,6 +833,18 @@ const basicCoins: BasicCoin[] = [
         address: '0xb706319d37b945727e71ae0d4353699d19112576',
         decimals: 18,
       },
+      [ChainId.MUM]: {
+        address: '0xeDb95D8037f769B72AAab41deeC92903A98C9E16',
+        decimals: 18,
+      },
+      [ChainId.ARBT]: {
+        address: '0xDC805eAaaBd6F68904cA706C221c72F8a8a68F9f',
+        decimals: 18,
+      },
+      [ChainId.OPTT]: {
+        address: '0x68Db1c8d85C09d546097C65ec7DCBFF4D6497CbF',
+        decimals: 18,
+      },
```

Comment:
We have used ARBT - Rinkeby based testnet before, and did not remove it yet. But the real Arbitrum Testnet that exists is based on Goerli and named `ARBG`. Same with Optimism, please use `OPTG`

Reply:
Of course